trashtalk/apps/cleanups/views/template_views.py

```diff
@@ -42,7 +43,18 @@ def cleanup_list(request):
 @login_required
 @parser_classes([FormParser])
 def cleanup_new(request):
-    context = {'form': CleanupFormSet()}
+    tool_categories = [] 
+
+    for category in ToolCategory.objects.filter(tool__isnull=False).distinct().order_by('name'):
+        tool_categories.append({
+            'name': category.name,
+            'tools': Tool.objects.filter(is_available=True, category=category).order_by('name')
+        })
+
+    context = {
+        'form': CleanupFormSet(),
+        'tool_categories': tool_categories
+    }
```

Comment:
I'm hesitant to place the tool request form into the cleanup creation view. 
1 . It can be an overwhelming amount of information for the user
2 . The user may not want tools or know how many tools s/he will need at first

Reply:
Fair enough. I'd like to discuss how this affects the userflow in the React App. Also, did we get a sense from the Public Works on whether previous tool requests could be revised/deleted, etc...?

Reply:
Public Works does have an ability to change tool requests. A rough summary of the process:
1. Users fill out a [tool request](https://docs.google.com/forms/d/e/1FAIpQLSfNHJV3Ht1U8k2Mq32VQqBsuAoenJElXB4Wpo2BCKW-LPLSMg/viewform?c=0&w=1)
2. Public Works Receives-
  a. A Google spreadsheet summary 
  b. An email with an attached [PDF Summary](https://drive.google.com/open?id=0B9_JW1qxM3VXdVYzN3Nva3NmWHhSdzMtYjA4Mi1uVnlSYmxv)
3. Users Receive-
  a. An email with the same summary
  b. A [link](https://docs.google.com/forms/d/e/1FAIpQLSfNHJV3Ht1U8k2Mq32VQqBsuAoenJElXB4Wpo2BCKW-LPLSMg/viewform?edit2=2_ABaOnufQFI_qebFMGzzi8W0Qu5j6WkE5Er-ihRbVRRuIfLsu3G5zANu1OIXA_54) to edit their response.

Public Works uses the spreadsheet as an inventory tool. It is the PDF that they really use to approve the tool request and organize their efforts. That's why it's so essential that we start our TrashTalk coordination with a PDF.